Source/Extensions/Blazorise.DataGrid/_DataGridCellDatePicker.razor.cs

```diff
@@ -13,18 +14,34 @@ namespace Blazorise.DataGrid;
 /// <typeparam name="TItem"></typeparam>
 public partial class _DataGridCellDatePicker<TItem> : ComponentBase
 {
+    #region Members
+
     protected string elementId;
 
     /// <summary>
     /// Value data type.
     /// </summary>
     private Type valueType;
 
+    private string dateDisplayFormat;
+
+    #endregion
+
+    #region Methods
+
     /// <inheritdoc/>
     protected override void OnInitialized()
     {
         valueType = Column.GetValueType( default );
         elementId = IdGenerator.Generate;
+
+        // this is a woraround for https://github.com/Megabit/Blazorise/issues/5837
+        // in 2.0 we need to remove formating as {0:dd.MM.yyyy}, and on support dd.MM.yyyy
+        if ( Column.DisplayFormat is not null && Column.DisplayFormat.StartsWith( "{0:" ) && Column.DisplayFormat.EndsWith( "}" ) )
```

Comment:
I really like `!String.IsNullOrEmpty`, thoughts?

Reply:
I don't think it is necessary here.